### PR TITLE
removing references to Mongo being Preview

### PR DESCRIPTION
**Pull request by tanberry**
Per Docs Issue #3028, removing references to Mongo being Preview

### CODE REVIEW

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/125-creating-the-prisma-schema.mdx [hunk 1]
```diff
@@ -17,19 +17,14 @@ datasource db {
   url      = env("DATABASE_URL")
 }
```
Comment (janpio) [inline content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/125-creating-the-prisma-schema.mdx:19]: I think you only want to reove the preview feature here:
```suggestion

generator client {
  provider = "prisma-client-js"
}
```

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/125-introspection.mdx [hunk 2]
```diff
@@ -46,11 +46,6 @@ datasource db {
   provider = "mongodb"
   url      = env("DATABASE_URL")
 }
-
-generator client {
```
Comment (janpio) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/125-introspection.mdx:50]: Same as https://github.com/prisma/docs/pull/3037/files#r842155051

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/125-introspection.mdx [hunk 3]
```diff
@@ -66,11 +61,6 @@ npx prisma db pull
 This command introspects our database and writes the inferred schema into your `prisma/schema.prisma` file:
 
 ```prisma file=prisma/schema.prisma
-generator client {
```
Comment (janpio) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/125-introspection.mdx:69]: Same as https://github.com/prisma/docs/pull/3037/files#r842155051

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/125-introspection.mdx [hunk 4]
```diff
@@ -93,11 +83,6 @@ model User {
 To be able to join data using the Prisma Client, you can add the [`@relation`](/reference/api-reference/prisma-schema-reference#relation) attributes to our models:
 
 ```prisma file=prisma/schema.prisma highlight=15;add|21;add
-generator client {
```
Comment (janpio) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/125-introspection.mdx:96]: Same as https://github.com/prisma/docs/pull/3037/files#r842155051

Removing any line here also means the `highlight` values need to be adapted.

File: content/200-concepts/200-database-connectors/07-mongodb.mdx [hunk 5]
```diff
@@ -149,12 +139,13 @@ const id = new ObjectId()
 
 ## Known limitations
```
Comment (janpio) [inline content/200-concepts/200-database-connectors/07-mongodb.mdx]: OT: Optimally we would present this differently now - these are "just" things done differently in MongoDB, not "limitations" of Prisma with MongoDB really.

File: content/400-reference/300-database-reference/01-database-features.mdx [hunk 6]
```diff
@@ -123,18 +123,18 @@ This section describes which database features exist on the NoSQL databases that
...
+| Transactions                              | ✔️                  |                                                                                                                                                                                              |
+| Indexes                                   | ✔️ with caveats     | Indexes can only be introspected if the field they refer to includes at least some data.                                                                                                     |
+| Autoincrementing IDs                      | No                  |                                                                                                                                                                                              |
+| Compound IDs                              | No                  | MongoDB does not support composite IDs (`@@id`)                                                                                                                                              |
```
Comment (janpio) [inline content/400-reference/300-database-reference/01-database-features.mdx]: Both are not supported on MongoDB, but the table is called "common MongoDB features and describes the level of support offered by Prisma" - that might be misleading or at least confusing.

File: content/400-reference/300-database-reference/01-database-features.mdx [hunk 7]
```diff
@@ -123,18 +123,18 @@ This section describes which database features exist on the NoSQL databases that
...
+| Arrays                                    | ✔️                  |                                                                                                                                                                                              |
+| Enums                                     | ✔️                  | Implemented at Prisma level                                                                                                                                                                  |
+| Native database types                     | ✔️                  | See: [Field mapping reference](/reference/api-reference/prisma-schema-reference#model-field-scalar-types)                                                                                    |
+| JSON support                              | ✔️                  | Advanced `Json` field filtering is supported by [Postgres and MySQL since 2.26.0](/concepts/components/prisma-client/working-with-fields/working-with-json-fields#filtering-on-a-json-field) |
```
Comment (janpio) [inline content/400-reference/300-database-reference/01-database-features.mdx]: Why does the PostgreSQL and MySQL support matter in a table about MongoDB?

**Review by janpio — COMMENTED** [review]
Left a few comments on where potentially too much was removed.

Would have expected more changes to be needed tbh 🤷

**Review by matthewmueller — APPROVED** [review]
Other than suggestions around keeping the generator, it looks good on my end!

**Comment by tanberry** [comment]
Thanks so much @janpio and @matthewmueller . Yes, perhaps there were some areas I removed the entire generator block when I should have just removed the line for `previewFeatures`, though some of them were not relevant. Others, it seems now, were. 

I can work on this more Tuesday, or @keerlu if you get to it before me feel free to edit as you see fit.

File: content/400-reference/300-database-reference/01-database-features.mdx (hunk 7 above)
Comment (tanberry) [reply content/400-reference/300-database-reference/01-database-features.mdx]: Excellent question, I missed that. Does MongoDB support `Json` field?

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/125-introspection.mdx (hunk 2 above)
Comment (keerlu) [reply content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/125-introspection.mdx:50]: Done

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/125-introspection.mdx (hunk 3 above)
Comment (keerlu) [reply content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/125-introspection.mdx:69]: Done

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/125-introspection.mdx (hunk 4 above)
Comment (keerlu) [reply content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/125-introspection.mdx:96]: Done

**Comment by keerlu** [comment]
I've fixed the generator block and found a few more places where the preview feature block needed removing.

File: content/400-reference/300-database-reference/01-database-features.mdx (hunk 7 above)
Comment (janpio) [reply content/400-reference/300-database-reference/01-database-features.mdx]: Yes. Not sure about the filtering etc though. @dpetrick ?

**Review by janpio — APPROVED** [review]

**Review by keerlu — APPROVED** [review]

File: content/400-reference/300-database-reference/01-database-features.mdx (hunk 6 above)
Comment (tanberry) [reply content/400-reference/300-database-reference/01-database-features.mdx]: I see the table as providing info about common features, and these two features are worth calling out and clearly noting that we don't support. In other words, the table is I think) meant to provide info on all common features, not just the ones we support.

File: content/400-reference/300-database-reference/01-database-features.mdx (hunk 7 above)
Comment (tanberry) [reply content/400-reference/300-database-reference/01-database-features.mdx]: @matthewmueller can you answer above question, about whether or not MongoDB supports `Jason` field?

File: content/400-reference/300-database-reference/01-database-features.mdx (hunk 6 above)
Comment (janpio) [reply content/400-reference/300-database-reference/01-database-features.mdx]: Then the description above the table is wrong. Then it is not "common MongoDB features and describes the level of support offered by Prisma".

File: content/200-concepts/200-database-connectors/07-mongodb.mdx (hunk 5 above)
Comment (tanberry) [reply content/200-concepts/200-database-connectors/07-mongodb.mdx]: good point, I changed "limitations" to considerations.

File: content/400-reference/300-database-reference/01-database-features.mdx (hunk 6 above)
Comment (tanberry) [reply content/400-reference/300-database-reference/01-database-features.mdx]: ok, will fix later today, thanks!

File: content/400-reference/300-database-reference/01-database-features.mdx (hunk 7 above)
Comment (tanberry) [reply content/400-reference/300-database-reference/01-database-features.mdx]: got the answer in slack thanks

**Pull request merged**